src/porepy/numerics/fv/biot.py

```diff
@@ -213,18 +208,22 @@ def assemble_matrix(self, g, data):
     def _discretize_flow(self, g, data):
 
         # Discretiztaion using MPFA
-        key = "flow"
+        key = self.flow_keyword
         md = pp.Mpfa(key)
 
         md.discretize(g, data)
 
     def _discretize_compr(self, g, data):
-        parameter_dictionary = data[pp.PARAMETERS]["flow"]
-        matrix_dictionary = data[pp.DISCRETIZATION_MATRICES]["flow"]
-        compr = parameter_dictionary["fluid_compressibility"]
-        poro = parameter_dictionary["porosity"]
-        matrix_dictionary["compr_discr"] = sps.dia_matrix(
-            (g.cell_volumes * compr * poro, 0), shape=(g.num_cells, g.num_cells)
+        """
+        TODO: Sort out time step (inconsistent with MassMatrix).
+        """
```

Comment:
In my opinion the time-step disappears from the MassMatrix

Reply:
I prefer to put it on the flow term.